limitador/Cargo.toml

```diff
@@ -54,6 +54,7 @@ base64 = { version = "0.22", optional = true }
 [dev-dependencies]
 serial_test = "3.0"
 criterion = { version = "0.5.1", features = ["html_reports"] }
+redis-test = { version = "0.4.0", features = ["aio"] }
```

Comment:
This broke the wasm build. It is really a dev-dependency (only for tests), so moved it here. 
If you need to declare a dependency that's only for a feature tho, but a runtime dep - like what you did here. There are two things you need to do: 
 - Mark it as `option = true`
 - Add it to the list of dependencies for that feature 

```toml
 We make redis and infinispan optional to be able to compile for wasm32.
[features]
default = ["disk_storage", "redis_storage"]
disk_storage = ["rocksdb"]
redis_storage = ["redis", "r2d2", "tokio"]
infinispan_storage = ["infinispan", "reqwest", "base64", "tokio"]
lenient_conditions = []
```